plugins/homekit/src/main.ts

```diff
@@ -111,6 +111,12 @@ export class HomeKitPlugin extends ScryptedDeviceBase implements MixinProvider,
             hide: true,
             description: 'The last home hub to request a recording. Internally used to determine if a streaming request is coming from remote wifi.',
         },
+        disableAutoAdd: {
```

Comment:
dont use negative names as variables. should be called `autoAdd`

Reply:
done